Misc/NEWS.d/next/Library/2019-07-18-14-39-59.bpo-37609.1ZYNbG.rst

```diff
@@ -0,0 +1 @@
+Add support for unc device path in :mod:`ntpath.splitdrive`
```

Comment:
Using the mod role here is not correct. Please see https://devguide.python.org/documenting/#id4.

Reply:
Thanks, I've updated it to use `meth`.

Reply:
It should be `func`.

Reply:
Thanks, I've updated it